### ISSUE TITLE
Error trying to set up kraber

### ISSUE THREAD

**Issue by 28development** [micheleriva/krabs#14]
I was trying to setup a simple kraber app but the setup fails.

I made the same steps you did in your tutorial video:

I get following error trying to start the server after setting up the index.js pages for the website-1 and website-2 subdomain.
The error occurs after running:

```node index.js```to start the server

```
internal/modules/cjs/loader.js:311
throw err;
      ^

Error: Cannot find module '/Users/ut/Documents/code/private/personax/personaimo/node_modules/krabs/dist/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:303:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:516:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:867:27)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/umuttufanoglu/Documents/code/private/personax/personaimo/index.js:3:15)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32) {
  code: 'MODULE_NOT_FOUND',
  path: '/Users/ut/Documents/code/private/personax/personaimo/node_modules/krabs/package.json',
  requestPath: 'krabs'
}
```

**Comment by Fellds** [comment]
Changing the package.json inside node_modules/krabs make it work to tests.

Change the `main` to `"main": "./dist/express-krabs/index.js"`

**Comment by 28development** [comment]
This worked somehow, but shouldn't this work "out of the box"?

**Comment by micheleriva** [comment]
Hi @28development and @Fellds , the problem has been fixed in version `0.1.2`. Please upgrade the krabs package to that version and it will work just fine!

Thanks for noticing the problem 🙂
